ehitchhiking-ui/src/app/pages/main-screen/main-screen-component/main-screen.component.css

```diff
@@ -0,0 +1,22 @@
+.btn-group {
```

Comment:
pls remove this file. For styles we should use files with .sass extension

Reply:
done, idk where they came from